python/jupyter_suggestions_rtc/package.json

```
@@ -0,0 +1,96 @@
+{
+  "name": "@jupyter/jupyter-suggestions-rtc",
```

Maybe we could also shorten the name of that package?

```suggestion
  "name": "@jupyter/suggestions-rtc",
```